src/arkhamdb/DeckImporterUi.ttslua

```diff
@@ -154,6 +154,17 @@ function whiteDeckChanged(_, _, inputValue) whiteDeckId = inputValue end
 
 function greenDeckChanged(_, _, inputValue) greenDeckId = inputValue end
 
+function updateDeckIds(idList)
+  greenDeckId = idList[1]
+  redDeckId = idList[2]
+  whiteDeckId = idList[3]
+  orangeDeckId = idList[4]
+  self.editInput({index=0, value=greenDeckId})
+  self.editInput({index=1, value=redDeckId})
+  self.editInput({index=2, value=whiteDeckId})
+  self.editInput({index=3, value=orangeDeckId})
+end
+
```

Comment:
Instead of using a table that depends on order, use a key-value-based table:
```
deckIDs = {
  white = "...",
  orange = "...",
  green = "...",
  red = "..."
}
```
Then you can get rid of LOC 158-161 too and just do:
`self.editInput({index=0, value=deckIDs.green})` etc.

Reply:
Totally agree the key-value table will be more legible, but I can't get rid of the 4 lines of code you mention because the `editInput` function does not trigger the input field's callback function (`colorDeckChanged` in this case), so the script will fail to load a deck because it isn't tracking the ID properly.  I could abstract that functionality out into a separate function if you think it would be clearer though.

Reply:
Ah that's a good point. Then just keep those lines, but I'd still turn it into a key-value table